src/Spout/Writer/Common/Manager/RegisteredStyle.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Box\Spout\Writer\Common\Manager;
+
+use Box\Spout\Common\Entity\Style\Style;
+
+class RegisteredStyle
```

Comment:
I think it's worth documenting what the style is about (as the logic is not trivial to understand)